pkg/crds/crds.go

```diff
@@ -0,0 +1,159 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package crds
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	k8sErrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var cephResources = []string{
+	"cephblockpoolradosnamespaces",
+	"cephblockpools",
+	"cephbucketnotifications",
+	"cephbuckettopics",
+	"cephclients",
+	"cephclusters",
```

Comment:
I'd suggest `cephclusters` should be first in the list. If the cluster is gone, other CRs should clean up more quickly and their finalizers may not even need to be forcefully removed. @BlaineEXE Any concern with that?